src/tracing/index.ts

```diff
@@ -93,13 +93,24 @@ export function startTracing(opts: Partial<Options> = {}): boolean {
   return true;
 }
 
-export function stopTracing() {
+export async function stopTracing() {
+  if (allowDoubleStart) {
+    isStarted = false;
+  }
+  // in reality unregistering is not reliable because of the function pointers
+  // floating around everywhere in the user code already and will lead to
+  // unexpected consequences should it be done more than once. We enable it
+  // mostly for tests.
```

Comment:
Curious of this note. Do you think this will be safe to call once in an application that will still be live after calling shutdown() for opentelemetry?

Reply:
If you do not intend to call `startTracing` later on in the process and are OK with having partially monkey-patched(that's how auto-instrumentations work in Node.js) versions of your dependencies, then you should be totally fine!

Do you have such a use-case?